lib/players/human.rb

```diff
@@ -0,0 +1,22 @@
+module Players
+    class Human < Player
+        def move board
```

Comment:
Separation of Concerns dictates that CLI logic like how to display board should be in its own CLI class.  All this should do is ask for the input, then take that input and apply it to your game logic.